docs/docsource/operation/XDM_GUI/file_operations/delete_files.rst

```diff
@@ -0,0 +1,88 @@
+.. _Delete Files:
+
+Delete Files
+============
+
+**Prerequisites:** An account with at least one bucket, containing at least one
+file.
+
+To delete files from a bucket:
+
+#. Click the **Data Browser** tab to open the bucket view.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser.png
+      :width: 100%
+
+#. Select an account from the account pull-down menu.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser_account_select.png
+      :width: 75%
+
+#. Click the link of the bucket you will delete from. 
+
+   .. image:: ../../Graphics/xdm_ui_bucket_select.png
+      :width: 75%
+
+#. The bucket's contents are displayed.
+
+   .. image:: ../../Graphics/xdm_ui_bucket_contents.png
+      :width: 75%
+
+#. Check the boxes next to the file or files you want to delete. To select all,
+   click the box next to **Name** above the file list.
+
+   .. image:: ../../Graphics/xdm_ui_file_delete_select.png
+      :width: 75%
+
+   The red **Delete** button brightens and becomes functional.	      
+
+#. |product| requests confirmation of the deletion.
+
+   .. image:: ../../Graphics/xdm_ui_file_delete_confirm.png
+      :width: 50%
+
+#. The file is deleted from the bucket.
+
+.. _Delete Versioned Files:
+
+Delete Versioned Files
+----------------------
+
+To delete a bucket, it must be completely empty. In a non-versioned bucket this
+task is straightforward, but for versioned buckets it can be difficult, because
+previous versions of the files may remain in the bucket. Follow this procedure
+to delete all versions of a file.
+
+**Prerequisites:** An account with at least one bucket, created with versioning
+enabled, containing at least one file.
+
+To delete a versioned file:
+
+#. Follow the steps in :ref:`Delete Files`. When you reach the bucket view,
```

Comment:
Does the user have to follow all the step procedure, or just step 1-4?

Reply:
I thought about this, but I was hesitant to include step numbers, because this UI remains in flux. By saying "When you reach the bucket view..." I step outside the numeration scheme and still preserve a usable navigation aid.

Reply:
Let's hope the UI won't change much... Otherwise, you'll probably have to redo all the screenshots 😢